index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="icon" href="img/icons8-blockchain.png" type="image/x-icon">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;1,100;1,300&display=swap"
+          rel="stylesheet">
+    <link rel="stylesheet " href="css/style.css" >
+    <title>Crypto Con &mdash; Your Blockchain Conference</title>
+</head>
+<body>
+<div class="hero">
+    <header id="header" class="nav">
+        <div class="nav__container">
+            <a class="nav__logo" href="index.html">
+                <img src="img/icons8-blockchain.png" alt="Website Logo">
+            </a>
+            <div class="nav__socials">
+                <div >
+                    <a href="https://twitter.com/CryptoCon2020" target="_blank" rel="noopener noreferrer">
+                        <img src="img/twitter.png" alt="Twitter"></a>
+                    <a href="https://www.facebook.com/CryptoCon2020/" target="_blank" rel="noopener noreferrer">
+                        <img src="img/facebook.png" alt="Facebook"></a>
+                    <a href="#"><p>English</p></a>
+                    <a href="#"><p>My Page</p></a>
+                    <a href="#"><p>Logout</p></a>
+                </div>
+            </div>
+            <nav>
+                <img id="toggle-nav" src="img/burger-bar.png" alt="Toggle Menu">
```

Comment:
- [ ] **Optional:** For a better UI, you could consider making the hamburger lighter when its background is dark. This is totally not required but my idea.

![image](https://user-images.githubusercontent.com/27709832/162810885-d14c48fe-a2b6-469f-bd93-818833bdf849.png)